README.md

```diff
@@ -7,3 +7,53 @@
 [![Try it on gitpod](https://img.shields.io/badge/try-on%20gitpod-brightgreen.svg)](https://gitpod.io/#https://github.com/PrismarineJS/prismarine-rng)
 
 Collection of utilities to work with minecraft RNG.
+
+## API
+
+## LCG
+?tbd
+
+## Random
+?tbd
+
+## crackPlayerSeed
+### crackPlayerSeed.crack(bits)
+Returns the decrypted player seed
+* `Returns` - Seed number
+* `bits` - Encrypted seed string?
+
+
+## enchantments
+### enchantments.findEnchantment(playerSeed, item, power, matching)
+* `Returns` - Object or null ?tbd
+* `playerSeed` - Number ?tbd
+* `item` - Item object ?tbd
+* `power` - ?tbd
+* `matching` - Matching function that returns a bool ?tbd
```

Comment:
Maybe it'll be easier to understand with an example:
```js
const playerSeed = 0x484b921be1c6n
const item = new Item(mcData.itemsByName.diamond_sword.id, 0)
console.log(Enchantments.findEnchantment(playerSeed, item, 15, (list) => {
  return list.find(e => e.enchant.name === 'sharpness' && e.level === 4) &&
         list.find(e => e.enchant.name === 'looting' && e.level === 3) &&
         list.find(e => e.enchant.name === 'sweeping' && e.level === 3) &&
         list.find(e => e.enchant.name === 'unbreaking' && e.level === 3)
}))
```
item is the object you want to enchant
power is the number of bookshelf arround the table
matching is a function that return true if the enchant is what you are looking for